projects/cashmere-examples/src/lib/typeahead-fetch/search.service.ts

```diff
@@ -23,7 +23,6 @@ export class SearchService {
             .pipe(
                 tap(response => {
                     this.itemsSubject.next(response.data.items);
-                    console.log(`fetched items for ${term}`);
```

Comment:
Thanks for cleaning that up. I recognize that I was the one that left it around by accident.